eng/pipelines/fluent_integration.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
     inputs:
       mavenPomFile: pom.xml
       goals: 'clean install'
-      options: '$(DefaultOptions) -P local,testFluent -T 1C -DskipTests'
+      options: '$(DefaultOptions) -P local -T 1C -DskipTests'
```

Comment:
Why do we remove this? I think ci.yaml for javagen still have `testVanilla` and `testAzure`

Reply:
This was moved further down the call chain. Fluent integration is now running like Javagen and TypeSpec where it runs the following pattern in CI

- Build JARs to generate code
- Generate code
- Run tests with generated code